cmd/manager/main.go

```diff
@@ -9,26 +9,33 @@ import (
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
 
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis"
+	app "github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/controller"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/logger"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	kubemetrics "github.com/operator-framework/operator-sdk/pkg/kube-metrics"
 	"github.com/operator-framework/operator-sdk/pkg/leader"
 	"github.com/operator-framework/operator-sdk/pkg/log/zap"
 	"github.com/operator-framework/operator-sdk/pkg/metrics"
+	"github.com/operator-framework/operator-sdk/pkg/restmapper"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
 	"github.com/spf13/pflag"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost       = "0.0.0.0"
-	metricsPort int32 = 8383
-	log               = logger.GetLogger("cmd")
+	metricsHost               = "0.0.0.0"
+	metricsPort         int32 = 8383
+	operatorMetricsPort int32 = 8686
```

Comment:
Very nice addition, but how the Operator Metrics is related to this migration?

Reply:
Actually, these variables are generated by the operator-sdk. Operator-sdk v0.9.0 added a new feature: `Generating and serving info metrics about each custom resource. By default these metrics are exposed on port 8686`.  Before v0.9.0, metrics for the operator are exposed via port 8383. In addition to that, in v0.9.0 custom resource metrics are also exposed, via port 8686. In the meantime, there is a breaking change: `ExposeMetricsPort is removed and replaced with CreateMetricsService() function`, which supports exposing multiple ports for the metrics service. Please find the release notes [here](https://github.com/operator-framework/operator-sdk/releases/tag/v0.9.0). Thanks.

Reply:
Hmm so I assume this should be implemented by other JIRA that aimed to upgrade to 0.9.0. As that JIRA was merged, I'll add a note about this one there.

Thanks for the detailed explanation, @xiezhang7